sdk/ai/azure-ai-projects/tests/agents/tools/multitool/test_agent_code_interpreter_and_function.py

```diff
@@ -0,0 +1,144 @@
+# pylint: disable=too-many-lines,line-too-long,useless-suppression
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+# cSpell:disable
+
+"""
+Multi-Tool Tests: Code Interpreter + Function Tool
+
+Tests various scenarios using an agent with Code Interpreter and Function Tool.
+All tests use the same tool combination but different inputs and workflows.
+"""
+
+import os
+import json
+import pytest
+from test_base import TestBase, servicePreparer
+from devtools_testutils import is_live_and_not_recording
+from azure.ai.projects.models import PromptAgentDefinition, CodeInterpreterTool, CodeInterpreterToolAuto, FunctionTool
+from openai.types.responses.response_input_param import FunctionCallOutput, ResponseInputParam
+
```

Comment:
Please make sure you run the 'black' tool to format the samples:

black --config ../../../eng/black-pyproject.toml .